src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -209,12 +207,29 @@ public long getPower() {
     @Override
     public void update() {
 
-        //container.givePower(1000,false);
-
-
+        //checking for the machine type
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "update" is 43 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)